newIDE/app/src/ResourcesList/LocalResourceSources.js

```diff
@@ -192,6 +197,18 @@ export default [
   },
 ];
 
+function _loadLastOpenedPath(): ?string {
+  const values = loadPreferencesValues();
+  if (values) return values.lastOpenedPath;
+  else return '';
+}
+
+function _saveLastOpenedPath(lastOpenedPath: string) {
+  let values = loadPreferencesValues();
+  values = { ...values, lastOpenedPath };
+  savePreferencesValues(values);
+}
```

Comment:
Can we avoid these functions? :)

Reply:
Oh Yes. We don't need these functions. I will remove them. :)